.gitlab-ci.yml

```
@@ -153,6 +154,9 @@ default:
     - if: $CI_COMMIT_REF_NAME =~ /^v[0-9]+\.[0-9]+.*$/              # i.e. v1.0, v2.1rc1
 
 .build-push-image:
+  extends:
+    - .kubernetes-env
+  image: quay.io/buildah/stable:v1.27
```

```suggestion
  extends:
    - .kubernetes-env
  image:                           ${BUILDAH_IMAGE}
```
This can be added here but you also need to adjust related publish jobs